benchmarks/bin/php_no_xdebug

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+temporaryPath="$(mktemp -t php.XXXX).ini"
+
+# Using awk to ensure that files ending without newlines do not lead to configuration error
+php -i | grep "\.ini" | grep -o -e '\(/[a-z0-9._-]\+\)\+\.ini' | grep -v xdebug | xargs awk 'FNR==1{print ""}1' > "$temporaryPath"
+
+php -n -c "$temporaryPath" -dopcache.enable_cli=1 -dzend.assertions=-1 "$@"
```

Comment:
How about just doing `-n` and not passing any config at all?

Reply:
hmm... probably `PDO` might not work, but let try :)